packages/core/src/Modules/Cashier/Components/side-note.jsx

```diff
@@ -10,21 +9,29 @@ const SideNoteBullet = ({ children }) => (
     </div>
 );
 
-const SideNote = ({ notes }) => (
+const SideNote = ({ notes, title, has_bullets = true }) => (
     <div className='cashier-side-note'>
         <DesktopWrapper>
-            <div className='cashier__header'>
-                <Localize i18n_default_text='Notes' />
-            </div>
+            <Text className='cashier-side-note__text' weight='bold' as='p'>
+                {title}
+            </Text>
         </DesktopWrapper>
-        {notes.map((note, i) => (
-            <SideNoteBullet key={i}>{note}</SideNoteBullet>
-        ))}
+        {notes.map((note, i) =>
+            has_bullets ? (
+                <SideNoteBullet key={i}>{note}</SideNoteBullet>
+            ) : (
+                <Text key={i} className='cashier-side-note__text' size='xxs' as='p'>
+                    {note}
+                </Text>
+            )
+        )}
     </div>
 );
 
 SideNote.propTypes = {
     notes: PropTypes.array.isRequired,
+    title: PropTypes.oneOfType([PropTypes.string, PropTypes.object]).isRequired,
```

Comment:
So we don't ever need to use `SideNote` component without `title` `prop`?

Reply:
confirmed with design team (dev), all side notes has title, and it might be different for different page 🙏

Reply:
but we can make it optional for flexibility 👍